### ISSUE TITLE
diploid option in variantCaller

### ISSUE THREAD

**Issue by ersgupta** [skoren/ArrowGrid#5]
I see that heterozygous variants option is disabled in consensus.sh (```DIPLOID="" # --diploid "```).

Any specific reason? I have a diploid genome.

**Comment by skoren** [comment]
The diploid option has never been recommended by PacBio. The most recent version of SMRTlink (5.1) supposedly has a better diploid option but we've never tested it so I  can't say how well it works.

**Comment by ersgupta** [comment]
ok. I am using v4.0, but get the following error (Sequel data):
```
[INFO] Will use arrow algorithm
[INFO] Found consensus models for: (P6-C4, S/P1-C1.1, S/P1-C1.2, S/P1-C1.3, S/P1-C1/beta, S/P2-C2)
Arrow: unsupported chemistries found: (unknown)
```

Have you encountered this in variantCaller?

**Comment by ersgupta** [comment]
Error solved by using v5.1 for variantCaller. I hope running previous steps with v4.0 and variantCaller with v5.1 would not be an issue.
